pkg/util/vnet/vnet.go

```diff
@@ -0,0 +1,41 @@
+package vnet
```

Comment:
none of pkg/util/vnet should be needed - it's just providing layers of wrapping that shouldn't be necessary